imagestreams/openad_workbench/imagestream.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: openad_workbench
+  namespace: redhat-ods-applications
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: >-
+      https://github.com/nerc-images/openad_workbench
+    opendatahub.io/notebook-image-name: >-
+      OpenAD Workbench
+    opendatahub.io/notebook-image-desc: >-
+	A Jupyter OpenShift AI Image with OpenAD for GPUs.
```

Comment:
@EldritchJS looks like there is a tab here instead of space.

Reply:
@computate Thanks for catching that. Changed it to space